test/luatest_helpers/vtest.lua

```diff
@@ -753,4 +753,5 @@ return {
     service_wait_for_error = service_wait_for_error,
     service_wait_for_new_error = service_wait_for_new_error,
     service_wait_for_activity = service_wait_for_activity,
+    wait_for_not_nil = wait_for_not_nil,
```

Comment:
The idea was to implement it also in this commit, not only export. I bet having it implemented but never used in one of the previous commits triggers luacheck (but CI luacheck works for the final version, not per commit, hence we don't see an error in CI).

Reply:
Fixed